using TaskException.Models;

Human ali =  new Human("Ali", "Taghizada", 21);
Console.WriteLine($"Name: {ali.Name} \nSurname: {ali.Surname} " +
    $"\nAge: {ali.Age}");


//Exception
//Human ali2 = new Human("Ali", "Taghizada", 0);
